src/default-feedback-prompt-handler.ts

```diff
@@ -0,0 +1,13 @@
+import { FeedbackPrompt, FeedbackPromptHandler } from "./types";
```

Comment:
would love a test of `defaultFeedbackPromptHandler` and another test that checks that `defaultFeedbackPromptHandler` is called by feedback mechanism if no user callback is assigned.